frame/mosaic-vault/src/mocks/runtime.rs

```diff
@@ -0,0 +1,258 @@
+}
+
+impl pallet_vault::Config for Test {
+    type Event = Event;
+	type Currency = Tokens;
+	type AssetId = MockCurrencyId;
+	type Balance = Balance;
+	type MaxStrategies = MaxStrategies;
+	type CurrencyFactory = LpTokenFactory;
+	type Convert = ConvertInto;
+	type MinimumDeposit = MinimumDeposit;
+	type MinimumWithdrawal = MinimumWithdrawal;
+	type PalletId = VaultPalletId;
+	type CreationDeposit = CreationDeposit;
+	type ExistentialDeposit = ExistentialDeposit;
+	type RentPerBlock = RentPerBlock;
+	type NativeAssetId = NativeAssetId;
+}
+
+parameter_types! {
+	pub const MinimumPeriod: u64 = MILLISECS_PER_BLOCK / 2;
+}
+
+impl pallet_timestamp::Config for Test {
+	type Moment = u64;
+	type OnTimestampSet = ();
+	type MinimumPeriod = MinimumPeriod;
+	
+    type WeightInfo = ();
+}
+
+parameter_types! {
+    pub const FeeFactor: Balance = 100;
+    pub const ThresholdFactor:Balance = 100;
+    pub const FeeAddress: u64 = FEE_ADDRESS as u64;
+    pub const MosaicVaultId: PalletId = PalletId(*b"test_pid");
+    pub const MaxFeeDefault: Balance = 10;
+    pub const MinFeeDefault: Balance = 0;
+}
+
+ord_parameter_types! {
+	pub const ADMIN: AccountId = ALICE;
+    pub const RELAYER: AccountId = RELAYER_ACCOUNT;
+}
+
+impl mosaic_vault::Config for Test {
+    type Event = Event;
+    type Currency = Tokens;
+    type Convert = ConvertInto;
+    type Balance = Balance;
+    type Nonce = Nonce;
+    type TransferDelay = TransferDelay;
+    type VaultId = VaultId;
+    type Vault = Vault;
+    type AssetId = MockCurrencyId;
+    type RemoteAssetId = MockCurrencyId;
+    type RemoteNetworkId = RemoteNetworkId;
+    type DepositId = DepositId;
+    type FeeFactor = FeeFactor;
+    type ThresholdFactor = ThresholdFactor;
+    type PalletId = MosaicVaultId;
+    type FeeAddress = FeeAddress;
+    type BlockTimestamp = Timestamp;
+    type MaxFeeDefault = MaxFeeDefault;
+    type MinFeeDefault = MinFeeDefault;
+    type RelayerOrigin = EnsureSignedBy<RELAYER, AccountId>;//<RootAccount, sp_core::sr25519::Public>;
+    type AdminOrigin = EnsureSignedBy<ADMIN, AccountId>;//<RootAccount, sp_core::sr25519::Public>;
+}
+
+pub struct ExtBuilder {
+	balances: Vec<(AccountId, MockCurrencyId, Balance)>,
+}
+
+impl Default for ExtBuilder {
+	fn default() -> Self {
+		Self { balances: vec![
+            (ALICE, MockCurrencyId::A, 1000000000),
+            (BOB, MockCurrencyId::A, 1000000000)
+                                       
+        ] }
+	}
+}
+
+impl ExtBuilder {
+	pub fn build(self) -> sp_io::TestExternalities {
+		let mut t = frame_system::GenesisConfig::default().build_storage::<Test>().unwrap().into();
+
+
+		orml_tokens::GenesisConfig::<Test> { balances: self.balances }
+			.assimilate_storage(&mut t)
+			.unwrap();
+
+            let mut ext = sp_io::TestExternalities::new(t);
+            ext.execute_with(|| System::set_block_number(1));
+            ext
+
+	}
+}
+
+// pub fn new_test_ext() -> sp_io::TestExternalities {
```

Comment:
please remove comment or comment why it is commented

Reply:
removed